components/backup/src/service.rs

```diff
@@ -56,33 +61,31 @@ where
         mut sink: ServerStreamingSink<CheckAdminResponse>,
     ) {
         let (tx, rx) = mpsc::unbounded();
-        match &self.router {
-            Some(router) => {
-                router.broadcast_normal(|| {
-                    PeerMsg::SignificantMsg(SignificantMsg::CheckPendingAdmin(tx.clone()))
-                });
-                let send_task = async move {
-                    let mut s = rx.map(|resp| Ok((resp, WriteFlags::default())));
-                    sink.send_all(&mut s).await?;
-                    sink.close().await?;
-                    Ok(())
-                }
-                .map(|res: Result<()>| match res {
-                    Ok(_) => {
-                        info!("check admin closed");
-                    }
-                    Err(e) => {
-                        error!("check admin canceled"; "error" => ?e);
-                    }
-                });
-                ctx.spawn(send_task);
+        if let Err(err) = self.snap_br_env.handle.broadcast_check_pending_admin(tx) {
```

Comment:
How long does it take? If it's too long, it might cause keep alive timeout.

Reply:
This will send a message to every region in the store. I guess that will be fast if there isn't a region blocking.